reo/scenario.py

```diff
@@ -231,16 +303,40 @@ def setup_pv(pv_dict, latitude, longitude, time_steps_per_hour):
             ModelManager.updateModel('WindModel', tmp, run_uuid)
             # TODO: remove the need for this db call by passing these values to process_results.py via reopt.jl
 
+
+        if inputs_dict["Site"]["CHP"].get("prime_mover") is not None or \
+                inputs_dict["Site"]["CHP"].get("max_kw", 0.0) > 0.0:
+            if boiler is not None:
+                steam_or_hw = boiler.existing_boiler_production_type_steam_or_hw
+            else:
+                steam_or_hw = 'hot_water'
+            chp = CHP(dfm=dfm, run_uuid=run_uuid,
+                      existing_boiler_production_type_steam_or_hw=steam_or_hw,
+                      oa_temp_degF=inputs_dict['Site']['outdoor_air_temp_degF'],
+                      site_elevation_ft=inputs_dict['Site']['elevation_ft'],
+                      time_steps_per_hour=inputs_dict.get('time_steps_per_hour'), **inputs_dict['Site']['CHP'])
+
+            # Update the model inputs if there are calculations with the tech class, CHP.py, etc
+            #tmp = dict()
+            # This is not currently updating anything because tmp is empty (hopefully not doing anything unintended)
+            #ModelManager.updateModel('CHPModel', tmp, run_uuid)
+
+
         util = Util(dfm=dfm,
                     outage_start_hour=inputs_dict['Site']['LoadProfile'].get("outage_start_hour"),
                     outage_end_hour=inputs_dict['Site']['LoadProfile'].get("outage_end_hour"),
                     )
-
+        ##[az] Imported from CHP, not sure if we need?
```

Comment:
not needed, legacy dat_file_manager method

Reply:
Thanks for confirming; this has been removed in commit ab8bee7f8e816c680d6eac52beb06527488afe0d